### PR TITLE
#54 : GiveofferController/Save the new/modified entities 

### PR DESCRIPTION
**Pull request by rameshthumati22**

### CODE REVIEW

File: src/main/java/com/kodekonveyor/work_request/offer/OfferValidationUtil.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.kodekonveyor.work_request.offer;
+
+import com.kodekonveyor.webapp.ValidationException;
+import com.kodekonveyor.work_request.WorkRequestConstants;
+
+public class OfferValidationUtil {
+
+  public static void inputValidation(final OfferDTO offerDTO) {
+    validatePrice(offerDTO);
```
Comment (szirbucz) [inline src/main/java/com/kodekonveyor/work_request/offer/OfferValidationUtil.java]: Makes no sense to have the validatePrice method. Inline it.

File: src/main/java/com/kodekonveyor/work_request/offer/OfferValidationUtil.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  public static void validatePrice(final OfferDTO offerDTO) {
+    if (0L == offerDTO.getPrice())
```
Comment (szirbucz) [inline src/main/java/com/kodekonveyor/work_request/offer/OfferValidationUtil.java]: You have a PMD rule violation here. Create a constant: ```private static long ZERO_PRICE = 0L``` to get rid of it.

**Comment by szirbucz** [comment]
Please implement save instead of validation, since it is in the task description.

**Comment by magwas** [comment]
updated with develop, please pull locally. You have PMD violations.

**Comment by magwas** [comment]
Closing due to inactivity. If you would like to work on it further, just grab the task again and reopen it.

**Pull request closed without merging**